ios/browser/api/ads/brave_ads.h

```diff
@@ -60,6 +60,9 @@ OBJC_EXPORT
 /// disabled.
 + (BOOL)shouldAlwaysRunService;
 
+/// Returns |true| if search result ads are supported.
```

Comment:
nit: preferred to use ' instead of |. | was the old Chromium way.

Reply:
agreed, but | is used throughout whole brave_ads.h file. So I think it is better to change it separately in whole file.